dashboard/db/migrate/20170927170055_add_timestamps_to_regional_partner.rb

```diff
@@ -0,0 +1,27 @@
+class AddTimestampsToRegionalPartner < ActiveRecord::Migration[5.0]
+  def up
+    add_and_backfill_timestamps_for(RegionalPartner)
+  end
+
+  def down
+    remove_timestamps_for(RegionalPartner)
+  end
+
+  private
+
+  def add_and_backfill_timestamps_for(model)
+    add_timestamps model.table_name, null: true
+    model.update_all(created_at: now, updated_at: now)
+    change_column_null model.table_name, :created_at, false
+    change_column_null model.table_name, :updated_at, false
+  end
+
+  def remove_timestamps_for(model)
+    remove_column model.table_name, :created_at
+    remove_column model.table_name, :updated_at
+  end
+
+  def now
+    @_now ||= Time.zone.now
+  end
+end
```

Comment:
This too. Migrations should be isolated in their own PRs for cleaner tracking and rollback.

Reply:
These 2 migrations could be combined (either in one migration, or 2 migrations in a single PR) but in any case they should be separate from the model changes.